### PR TITLE
Release 0.1.4 (sanitizer) and 0.2.1 for sanitizer_macros

### PR DESCRIPTION
**Pull request by Daksh14**

### CODE REVIEW

File: sanitizer-macros/tests/struct.rs [hunk 1]
```diff
@@ -38,7 +42,9 @@ fn sanitizer_check() {
         upper_case: String::from("hello, world"),
         camel_case: String::from("hello_world"),
         snake_case: String::from("helloWorld"),
-        screaming_snake_case: String::from("helloWorld"),
+        screaming_kebab_case: String::from("helloWorld"),
+        kebab_case: String::from("hello-World"),
+        screaming_snake_case: String::from("HELLO-WORLD"),
```
Comment (RabidFire) [inline sanitizer-macros/tests/struct.rs]: These test cases aren't great - Example: "hello-World" -> "hello-world" can be addressed by lower_case as well. You should do "Hello, World" -> "hello-world" which considers capital letters & punctuation.

File: sanitizer-macros/tests/struct.rs [hunk 2]
```diff
@@ -54,6 +60,9 @@ fn sanitizer_check() {
     assert_eq!(instance.snake_case, "hello_world");
     assert_eq!(instance.screaming_snake_case, "HELLO_WORLD");
     assert_eq!(instance.clamp_str, "Hello, Wor");
+
```
Comment (RabidFire) [inline sanitizer-macros/tests/struct.rs]: Remove newline here?

File: .gitignore [hunk 3]
```diff
@@ -86,3 +86,7 @@ bh_unicode_properties.cache
 # Sublime-github package stores a github token in this file
 # https://packagecontrol.io/packages/sublime-github
 GitHub.sublime-settings
+
+
+## Jetbrains
+.idea/
```
Comment (RabidFire) [inline .gitignore]: Add EOF line here?

File: sanitizer-macros/src/lib.rs [hunk 4]
```diff
@@ -67,13 +69,16 @@ mod type_ident;
...
+/// - **kebab_case**: Convert input to kebab case.
+/// - **screaming_kebab_case**: Convert input to shouty kebab case.
 /// - **e164**: Convert a valid phone number to the e164 international standard, panic if invalid phone number.
 /// - **clamp(min, max)**: Limit an integer input to this region of min to max.
 /// - **clamp(max)**: Cut the string if it exceeds max.
 /// - **screaming_snake_case**: Convert input to screaming snake case.
 /// - **custom(function)**: A custom function that is called to sanitize a field
 /// according to any other way.
 #[proc_macro_derive(Sanitize, attributes(sanitize))]
+#[allow(unused_assignments)]
```
Comment (RabidFire) [inline sanitizer-macros/src/lib.rs]: What's the unused_assignments for

File: sanitizer-macros/tests/struct.rs (hunk 1 above)
Comment (Daksh14) [reply sanitizer-macros/tests/struct.rs]: Switched to `hello, world`

**Review by RabidFire — APPROVED** [review]

**Pull request merged**